packages/oauth/oauth-provider/src/oauth-provider.ts

```diff
@@ -134,7 +134,7 @@ export {
 export type RouterOptions<
   Req extends IncomingMessage = IncomingMessage,
   Res extends ServerResponse = ServerResponse,
-> = {
+> = DeviceManagerOptions & {
```

Comment:
Allows configuring the `DeviceManager` options when creating a the provider's middleware.